scss/grid/_grid.scss

```diff
@@ -33,6 +33,9 @@ $block-grid-max: 8 !default;
 // Internal value to store the end column float direction
 $-zf-end-float: if($grid-column-align-edge, $global-right, $global-left);
 
+// Extend column with this alias if no alias set to false
```

Comment:
Please add a sass-doc documentation (see the comments above, or [this reference](sassdoc.com/annotations/)).

Also, I think we should avoir relative descriptors like `this`, and use a descriptive formulation. i.e:
```
// Selector used for an alias of column (with @extend). If `false`, no alias is created.
```